### DISCUSSION TITLE
Edit quotes fields

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14746] [How to]
I would to add/edit/ the fields/columns and calculations that appears in the quote panel.
How to customize this part?

![Schermata 2021-05-30 alle 09 56 37](https://user-images.githubusercontent.com/85027316/120096752-855bb100-c12d-11eb-9827-201eb4fdccc9.png)

**Comment by vovpff** [comment]
Admin panel - modules - edit fields.

Walk trough admin panel at first.

**Reply by espe808** [reply]
Thanks.
I would to replace the item name with the contacts of that company.
Do you think I am in the right way?

**Reply by vovpff** [reply]
Not so. Item name for advanced modules. Projects is basic type

**Reply by espe808** [reply]
And so could I replicate a quote module like this with the contacts instead the items?
And i could show this quote module in the accounts window?

**Reply by vovpff** [reply]
Yes, you can change nodule type to advanced and can use contacts as line items.

Quotes should be visible as default.

There are a lot of ways how to achieve your task but no ability to descibe them because a lot of time needed.

**Reply by espe808** [reply]
I have found a solution! :)
Now, how could to have an immediate field calculation like "Net" field in the quote module?

**Reply by vovpff** [reply]
I know only one way. Js coding.

**Reply by espe808** [reply]
Where I can add Js coding?
In the Admin Panel?

**Reply by vovpff** [reply]
No, you need to modify original code. This could be dangerous and will affect to whole system.

**Reply by espe808** [reply]
> Yes, you can change nodule type to advanced and can use contacts as line items.
> 
> Quotes should be visible as default.

Ok. And to do this?

**Reply by vovpff** [reply]
Admin panel - standard modules - edit fields

**Reply by espe808** [reply]
I have done!

![Schermata 2021-05-30 alle 13 50 21](https://user-images.githubusercontent.com/85027316/120103082-3eca7e80-c14e-11eb-952a-8b906923ccaa.png)

Now how I could display this table in a new account module=?

**Reply by noelhunter** [reply]
I have done a similar thing with a custom module.  Is there a way to populate the Unit Price field in a quote when adding an item from a custom type?  I have tried adding a unit_price field in my module, but it doesn't carry over the value when I add an item to a quote.  The Item Name field is populated, but not the unit price.

**Reply by vovpff** [reply]
Hi. I think what original js doesn't work with custom modules. There is a lot of limitations what you cant see
